impl/src/main/java/slack/ThreadMessageHandler.java

```diff
@@ -61,10 +61,10 @@ public boolean shouldHandle(String type, JsonObject body) {
 
                 //No thread found -> create new
                 .onErrorResumeNext(throwable -> {
-                    if (throwable instanceof WebException
-                        && NOT_FOUND.equals(((WebException)throwable).getStatus())) {
-                        return createQuestionAndPostToSlack(message);
-
+                    if (throwable instanceof WebException) {
```

Comment:
There are sonar rules for not nesting ifs, but I guess we're not using them. 😄

Reply:
fixed